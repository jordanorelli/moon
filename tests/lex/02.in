-1
1
+1

-0
0
+0

-.1
.1
+.1

-0.1
0.1
+0.1

-1.5
1.5
+1.5

-0e9
0e9
+0e9

-1.2e5
1.2e5
+1.2e5

-1.2e-5
1.2e-5
+1.2e-5

0xfacebeef
0723414
