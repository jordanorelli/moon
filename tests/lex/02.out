{t_real_number -1}
{t_real_number 1}
{t_real_number +1}
{t_real_number -0}
{t_real_number 0}
{t_real_number +0}
{t_real_number -.1}
{t_real_number .1}
{t_real_number +.1}
{t_real_number -0.1}
{t_real_number 0.1}
{t_real_number +0.1}
{t_real_number -1.5}
{t_real_number 1.5}
{t_real_number +1.5}
{t_real_number -0e9}
{t_real_number 0e9}
{t_real_number +0e9}
{t_real_number -1.2e5}
{t_real_number 1.2e5}
{t_real_number +1.2e5}
{t_real_number -1.2e-5}
{t_real_number 1.2e-5}
{t_real_number +1.2e-5}
{t_real_number 0xfacebeef}
{t_real_number 0723414}
